tsconfig.eslint.json

```
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "module": "es6",
+    "target": "es2019",
```

Should the target be es2021, like in eslint.config.mjs? Or even es2024 in both places (see [this](https://github.com/eslint/eslint/pull/17328))?
```suggestion
    "target": "es2021",
```